models/config_dump.go

```diff
@@ -0,0 +1,398 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	adminapi "github.com/envoyproxy/go-control-plane/envoy/admin/v3"
```

Comment:
There are a lot of imports of core features of envoy for bringing a "model".
Perhaps on this case it's safe, but also consider clonning the struct if this could bring extra non-necessary imports.